src/amp-story-player/amp-story-player-impl.js

```diff
@@ -1591,6 +1595,19 @@ export class AmpStoryPlayer {
     return index >= this.stories_.length || index < 0;
   }
 
+  /** @private */
+  initializeAutoplay_() {
```

Comment:
This is good now, but if you have many behavior configurations, it might be good to apply all the configurations in the same place instead of having one method per parameter right?

Reply:
Yea, we already have several behavior configurations. Applying all the configurations in one place would still need to call to each individual method since they all have different combinations, but we can definitely clean this up if it becomes annoying.

Reply:
Yeah if it becomes annoying the player could have a default configuration that gets merged with the JSON passed and you fetch the values from there to modify the behaviors